sysl2/sysl/Parser_test.go

```diff
@@ -13,39 +14,46 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func pyParse(filename, root string) (*sysl.Module, error) {
-	output := filename + ".pb"
+func readSyslModule(filename string) (*sysl.Module, error) {
+	var buf bytes.Buffer
 
-	args := []string{"pb", "-o", output, filename}
+	f, err := os.Open(filename)
+	if err != nil {
+		return nil, err
+	}
+	io.Copy(&buf, f)
+	f.Close()
+
+	module := &sysl.Module{}
+	if err := proto.UnmarshalText(buf.String(), module); err != nil {
+		return nil, err
+	}
+	return module, nil
+}
+
+func pySysl() string {
+	if pySysl, found := os.LookupEnv("SYSL_PYTHON_BIN"); found {
+		return pySysl
+	}
+	return "sysl"
+}
+
+func pyParse(filename, root, output string) (*sysl.Module, error) {
+	args := []string{"textpb", "-o", output, filename}
 	if len(root) > 0 {
 		rootArg := []string{"--root", root}
-		// TODO: This looks dubious
-		args[2] = root + "/" + args[2]
 		output = args[2]
```

Comment:
Remove?

Reply:
Sorry, I must be having a completely dumb moment, but isn't `output` already equal to `args[2]`

Reply:
Ah yes, I see now. Off-by-one error in my brain.